assets/js/components/CheckResultsOverview/CheckResultsOverview.jsx

```diff
@@ -29,10 +33,24 @@ function CheckResultsOverview({
     );
   }
 
-  if (error) {
+  if (error || catalogError) {
     return (
       <div className="flex flex-col items-center mt-2 px-4">
-        <div className="text-center text-xs">{error}</div>
+        <div className="text-center text-xs">{error || catalogError}</div>
+      </div>
+    );
+  }
+
+  if (!catalogData?.length) {
+    return (
+      <div className="flex flex-col items-center h-full">
+        <h1 className="text-center text-2xl font-bold">Check Results</h1>
+        <h6 className="opacity-60 text-xs">Checks coming soon</h6>
```

Comment:
Bringing focus to the wording of the message here, thoughts?

Reply:
Currently in the cluster detail, and specifically for ASC/ERS clusters we have `Coming soon for ASCS/ERS`.

To me the thing is: 
we either do an effort to detect a meaningful user-friendly name of the target 
- Coming soon for Hosts
- Coming soon for Clusters
- Coming soon for ASCS/ERS clusters

Or we use a generic message.

The generic message, imho, would be good enough to start with,